docs/ImageInput.md

```diff
@@ -5,53 +5,149 @@ title: "The ImageInput Component"
 
 # `<ImageInput>`
 
-`<ImageInput>` allows to upload some pictures using [react-dropzone](https://github.com/okonet/react-dropzone).
+`<ImageInput>` allows editing and uploading iumages (png, jpg, gif, etc). It is powered by [react-dropzone](https://github.com/okonet/react-dropzone).
 
 ![ImageInput](./img/image-input.png)
 
-## Properties
+## Usage
+
+```jsx
+import { ImageInput, ImageField } from 'react-admin';
+
+<ImageInput source="pictures" label="Related pictures">
+    <ImageField source="src" title="title" />
+</ImageInput>
+```
+
+`<ImageInput>` uses its child component to give a preview of the files. `<ImageInput>` renders it child once per file, inside a `<RecordContext>`, so the child can be a Field component. The default [`<ImageField>`](./ImageField.md) renders the name of the file(s), with a hyperlink.
```

Comment:
don't think there is an hyperlink, only the image is displayed I believe

Reply:
Right, fixed